### PR TITLE
[op docs] Remove konvoy download instructions from kommander docs

### PR DESCRIPTION
**Pull request by gracedo**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-62854

## Description of changes being made
Looks like konvoy docs do not include specific details on download instructions, so we should remove them from kommander and just link to the konvoy docs.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
@gracedo this was done on purpose bc Chris Gaun requested. Why do we want to take it out now?

File: pages/dkp/kommander/1.3/install/index.md [hunk 1]
```diff
@@ -10,50 +10,22 @@ excerpt: Getting started with Kommander
 Kommander is a tool that provides a command console for deploying, monitoring, and managing production-ready Kubernetes clusters on an enterprise scale. Kommander supports both Konvoy and non-Konvoy clusters.
 
 # Before you begin
-Prerequisites for Kommander are the same as those for Konvoy. See the [Konvoy Prerequisites](/dkp/konvoy/1.6/quick-start/#prequisites) for information.
+Prerequisites for Kommander are the same as those for Konvoy. See the [Konvoy Prerequisites](/dkp/konvoy/1.7/quick-start/#prequisites) for information.
 
 # Download and Install
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/install/index.md:15]: This needs some rewording.

**Review by cmastr — CHANGES_REQUESTED** [review]

**Comment by gracedo** [comment]
> @gracedo this was done on purpose bc Chris Gaun requested. Why do we want to take it out now?

I wasn't aware of Chris's request. Do you know why he requested it be documented this way? Maybe Konvoy at the time had the same detailed instructions? I just find it strange to include these instructions when Konvoy instructs the user to check their email for the download how-to. @dkoshkin wdyt?

**Review by cneth — APPROVED** [review]

**Comment by cmastr** [comment]
> > @gracedo this was done on purpose bc Chris Gaun requested. Why do we want to take it out now?
> 
> I wasn't aware of Chris's request. Do you know why he requested it be documented this way? Maybe Konvoy at the time had the same detailed instructions? I just find it strange to include these instructions when Konvoy instructs the user to check their email for the download how-to. @dkoshkin wdyt?

yeah, sorry I was a little hasty. I think you're right that this used to work differently and maybe doesn't need to be this way anymore. Let's get another opinion though. Will wait for Dimitri.

**Review by dkoshkin — APPROVED** [review]

**Comment by gracedo** [comment]
@cmastr I synced with @dkoshkin and he agrees that we should remove these instructions from kommander docs and link to konvoy download docs

**Comment by cmastr** [comment]
Ok, good to go then. Thanks Grace!

**Review by cmastr — APPROVED** [review]

**Pull request merged**